common/manifest.yaml

```diff
@@ -0,0 +1,14 @@
+image: __IMAGE_REPO__/__IMAGE_NAME__:__RELEASE_TAG__
```

Comment:
we don't need this file.

Reply:
@chenzhiwei why does not need this?

Reply:
this file is used to build multiarch image, but we use a command that does same:

> manifest-tool push from-args --platforms linux/amd64,linux/ppc64le,linux/s390x --template $(REGISTRY)/$(IMG)-ARCH:$(VERSION) --target $(REGISTRY)/$(IMG) --ignore-missing